### ISSUE TITLE
can't get minute mode to work (still)

### ISSUE THREAD

**Issue by auwsom** [quantopian/zipline#2383]
Dear Zipline Maintainers,

using 2.7 base conda environment in Jupyter Lab and command line
have daily algos working fine
using basic algo that just records data: record(QQQ  = data.current(context.asset, "close"))

# Description of Issue

As the title says, "can't get minute mode to work (still)", and I've tried everything I could to get different results. Similar to here: https://github.com/quantopian/zipline/issues/1014, and elsewhere.

I finally thought I overcame the benchmark issue by copying the same minute data into SPY_benchmark.csv. That threw no errors, is not attempting to ingest benchmarks (so I'm assuming it is using my replaced ones), but I'm still getting a daily output perf.

Is the minute testing not working in general, or are there others who have it working? 

Thank you

>zipline run -f mtest.py --start '2018-12-03' --end '2018-12-07' --data-frequency minute

C:\Users\username\Anaconda35\lib\site-packages\empyrical\stats.py:704: RuntimeWarning: invalid value encountered in true_divide  out=out,
C:\Users\username\Anaconda35\lib\site-packages\empyrical\stats.py:790: RuntimeWarning: invalid value encountered in true_divide  np.divide(average_annual_return, annualized_downside_risk, out=out)
[2018-12-10 05:36:29.606680] INFO: zipline.finance.metrics.tracker: Simulated 5 trading days
first open: 2018-12-03 14:31:00+00:00
last close: 2018-12-07 21:00:00+00:00
                           QQQ  algo_volatility  algorithm_period_return  \
2018-12-03 21:00:00+00:00  NaN              NaN                      0.0
2018-12-04 21:00:00+00:00  NaN              0.0                      0.0
2018-12-05 21:00:00+00:00  NaN              0.0                      0.0
2018-12-06 21:00:00+00:00  NaN              0.0                      0.0
2018-12-07 21:00:00+00:00  NaN              0.0                      0.0       .............................

**Comment by CapitalZe** [comment]
I'm in the same position. I can successfully ingest my custom minute data, but no sqlite file is generated.

I think that is the bulk of the problem. It then always fails or refers elsewhere to run the back test.

**Comment by CapitalZe** [comment]
Any progress made on this?

**Comment by CapitalZe** [comment]
May I ask your source for custom data? Was it Duckascopy?

I am using it from there (with some trials with finam data) and facing the exact same issue.

I think it is something to do with pandas not being able to convert the string to a timestamp. If you run a `!zipine bundle` (if using Jupyter NB) you will likely see the throwback: `ValueError: could not convert string to Timestamp`

I think this means the dates are being parsed correctly by the time data is not, hence the `no minute data for sid`, not anything to do with the benchmark data per se. This is speculation though.

If anyone has any input on this, please share. It's a nightmare trying to get minute data to work.

I also got fairly close using Sentdex tutorial wherein he relies on Pandas panels, but this method has me closer.

**Comment by rajach** [comment]
Did you find a solution for this?
First I was getting a KeyError: Timestamp('2019-11-29 13:43:00+0000', tz='UTC') and I loaded the file to dataframe applied tz_localize(tz='US/Eastern') and saved it back. Then the error changed to loading zero percent.

**Comment by CapitalZe** [comment]
No, I am getting a KeyError: Timestamp('1990-01-02 00:00:00+0000', tz='UTC')
 What is strange is that I do not reference that date at all, and I cannot trace where it is. My date ranges are flagged as 2017 to 2020.

Sorry I cannot be of more help.

**Comment by cheesechoi** [comment]
Hi all. Have you guys solve this prob ? I've got this and struggled whole this week.
